src/pages/new-chat/new-chat.ts

```diff
@@ -9,22 +12,30 @@ import { User } from 'api/models/whatsapp-models';
   templateUrl: 'new-chat.html'
 })
 export class NewChatComponent implements OnInit {
-  users;
+  searchPattern = '';
```

Comment:
It is necessary to initialize this field as an empty string, otherwise the search bar will show `undefined` as we open the page for the first time.